backend-schedule/src/main/java/eu/socialedge/hermes/backend/schedule/domain/api/ScheduleGenerationApi.java

```diff
@@ -17,51 +17,45 @@
 import eu.socialedge.hermes.backend.schedule.domain.BasicScheduleGenerator;
 import eu.socialedge.hermes.backend.schedule.domain.ScheduleSpecification;
 import eu.socialedge.hermes.backend.schedule.domain.repository.ScheduleRepository;
-import eu.socialedge.hermes.backend.transit.domain.Route;
-import eu.socialedge.hermes.backend.transit.domain.repository.RouteRepository;
+import eu.socialedge.hermes.backend.transit.domain.repository.LineRepository;
 import lombok.val;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.rest.webmvc.RepositoryRestController;
 import org.springframework.http.ResponseEntity;
+import org.springframework.util.AntPathMatcher;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.util.UriComponentsBuilder;
 
 import javax.validation.Valid;
 import javax.validation.constraints.NotNull;
 
-import java.util.Optional;
-
 import static org.springframework.web.bind.annotation.RequestMethod.POST;
 
-@RestController
-@RequestMapping(path = "/schedules")
+@RepositoryRestController
```

Comment:
Its a regular rest controller, why not?
Here is how you can disable particular spring data rest method. http://stackoverflow.com/a/29280226/2180005
In our case we should override `save` method in ScheduleRepository and annotate it with @RestResource(exported = false). You can create regular controller than. 

Maybe you meant a request mapping path clashing? Is there a problem with that?